crates/sui-graphql-rpc/schema/draft_schema.graphql

```diff
@@ -596,10 +596,8 @@ type TransactionBlock {
 }
 
 enum TransactionBlockKindInput {
-  CONSENSUS_COMMIT_PROLOGUE
-  GENESIS
-  CHANGE_EPOCH
```

Comment:
Where should people turn to if they'd like to query for these transactions?

Reply:
These are all different kinds of system transaction, so people can still find them by using the `SYSTEM_TX` variant, they will just need to do some client-side filtering if they want to differentiate between genesis, change epoch, consensus commit etc.